tests/testthat/test-matrix_form.R

```diff
@@ -1,8 +1,11 @@
 test_that("matrix_form works with and without indentation", {
-  iris_output <- iris %>%
-    group_by(Species) %>%
-    summarize("all obs" = round(mean(Petal.Length), 2)) %>%
-    mutate("Petal.Length" = "Mean")
+  skip_if_not_installed("dplyr")
+
+  iris_output <- dplyr::summarize(dplyr::group_by(iris, Species),
+    "all obs" = round(mean(Petal.Length), 2)
+  )
+  iris_output <- dplyr::mutate(iris_output, "Petal.Length" = "Mean")
+
```

Comment:
FYI: `dplyr` re-exports `%>%` so you can safely use it after checkinf that `dplyr` is available

Reply:
true